GraphConsoleAppV3/GraphRequests.cs

```diff
@@ -0,0 +1,1534 @@
+﻿#region
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.IO;
```

Comment:
done

---
In reply to: [80540806](https://github.com/Azure-Samples/active-directory-dotnet-graphapi-console/pull/37#discussion_r80540806) [](ancestors = 80540806)